client/src/components/DataFiles/DataFilesSidebar/DataFilesSidebar.js

```diff
@@ -86,11 +88,12 @@ const DataFilesSidebar = ({ readOnly }) => {
                   <i className="icon-folder" /> Folder
                 </span>
               </DropdownItem>
-              {systems.some(s => s.scheme === 'projects') && (
-                <DropdownItem onClick={toggleAddProjectModal}>
-                  <i className="icon-folder" /> Shared Workspace
-                </DropdownItem>
-              )}
+              {systems.some(s => s.scheme === 'projects') &&
```

Comment:
Let's refactor this `systems.some` call out, since we now have a `sharedWorkspaces` object above on [line 37](task/FP-1118-a2cps-no-datafiles-systems). We can instead look to make sure `sharedWorkspaces` is not null